### PR TITLE
[MariaDB compatibility] Update ChannelDataGrid.php

### PR DESCRIPTION
**Pull request by helgvor-stoll**
The channel datagrid is now MariaDB compatible

## Issue Reference
# #3

### CODE REVIEW

**Comment by navneetbhardwaj** [comment]
@helgvor-stoll 

Thank you for your contribution. It looks like the linting and code style tests have failed. To ensure code quality, please run the following commands:

1. **Linting and Code Style Fixes**:

    ```bash
    vendor/bin/pint
    ```

   This will analyze and automatically fix any style issues in your PHP code.

2. **Running Tests**:

    ```bash
    vendor/bin/pest
    ```

   This will execute your test suite to ensure everything is working as expected.

Please let me know if you encounter any issues or need further assistance!

**Comment by navneetbhardwaj** [comment]
@helgvor-stoll

Thank you for your contribution. It looks like the linting and code style tests have failed. To ensure code quality, please run the following commands:

1. **Linting and Code Style Fixes**:

    ```bash
    vendor/bin/pint
    ```

   This will analyze and automatically fix any style issues in your PHP code.

2. **Running Tests**:

    ```bash
    vendor/bin/pest
    ```

   This will execute your test suite to ensure everything is working as expected.

Please let me know if you encounter any issues or need further assistance!

**Comment by helgvor-stoll** [comment]
@navneetbhardwaj Normally it's ok now. I will improve my knowledges in Github for future.

File: packages/Webkul/Admin/src/DataGrids/Settings/ChannelDataGrid.php [hunk 1]
```diff
@@ -49,7 +50,6 @@ public function prepareQueryBuilder()
 
         return $queryBuilder;
     }
-
```
Comment (navneetkumar-pim-webkul) [inline packages/Webkul/Admin/src/DataGrids/Settings/ChannelDataGrid.php]: @helgvor-stoll please add the new line here

Comment (helgvor-stoll) [reply packages/Webkul/Admin/src/DataGrids/Settings/ChannelDataGrid.php]: I changed it. This time, I copied all the file

**Pull request merged**